docs/configure_https.md

```diff
@@ -54,7 +54,7 @@ Copy the template **nginx.https.conf** as the new configuration file:
 ```
   cp nginx.https.conf nginx.conf
 ```
-Edit the file nginx.conf and replace two occurrences of **harbordomain.com** to your own host name, such as reg.yourdomain.com .
+Edit the file nginx.conf and replace two occurrences of **harbordomain.com** to your own host name, such as reg.yourdomain.com . Replace the port "443" in line "rewrite ^/(.*) https://$server_name:443/$1 permanent;" as well if you use a customized port rather than the default one.  
```

Comment:
If you use a customized port rather than the default port 443, replace the port "443" in the line "rewrite ^/(.*) https://$server_name:443/$1 permanent;" as well. Please refer to the installation guide for other required steps of port customization.